tests/by-util/test_ls.rs

```diff
@@ -333,6 +333,61 @@ fn test_ls_long() {
     }
 }
 
+#[test]
+fn test_ls_long_format() {
+    #[cfg(not(windows))]
+    let last;
+    #[cfg(not(windows))]
+    {
+        let _guard = UMASK_MUTEX.lock();
+        last = unsafe { umask(0) };
+
+        unsafe {
+            umask(0o002);
+        }
+    }
+
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    at.mkdir(&at.plus_as_string("test-long-dir"));
+    at.touch(&at.plus_as_string("test-long-dir/test-long-file"));
+    at.mkdir(&at.plus_as_string("test-long-dir/test-long-dir"));
+
+    for arg in &["-l", "--long", "--format=long", "--format=verbose"] {
+        let result = scene.ucmd().arg(arg).arg("test-long-dir").succeeds();
```

Comment:
@miDeb we should still fix these warnings :)

Reply:
They're because the variable result is unused. I will fix them while making the test for the feature I'm currently working on (coloring of symlinks in`ls -l`).

I'm done with the feature and making the tests right now, then will make a pull request for it. I think it should be okay for the fix for this result variable being unused thing to be in the same pull request.